fetch.js

```diff
@@ -50,7 +50,7 @@ function normalizeName(name) {
     name = String(name)
   }
   if (/[^a-z0-9\-#$%&'*+.^_`|~!]/i.test(name) || name === '') {
-    throw new TypeError('Invalid character in header field name')
+    throw new TypeError('Invalid character in header field name: ' + name)
```

Comment:
If the value of `name` is `''` the error message would say:
>Invalid character in header field name: 

If we add quotes around it like this
```suggestion
    throw new TypeError('Invalid character in header field name: "' + name + '"')
```
Then the error message for an empty header name would say:
>Invalid character in header field name: ""

What do you think @CarmeloPatti?

Reply:
very good suggestion, thanks :) I pushed it already